build/dependencies.props

```diff
@@ -14,6 +14,8 @@
     <NpgsqlVersion>3.2.0-*</NpgsqlVersion>
     <NpgsqlEntityFrameworkCorePostgreSQLVersion>1.*</NpgsqlEntityFrameworkCorePostgreSQLVersion>
 
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
But, just realized we don't need to set `$(DapperVersion)`, `$(NpgsqlVersion)`, or `$(NpgsqlEntityFrameworkCorePostgreSQLVersion)` at all. Removing the last two will bring everything together.